modules/gateway/gateway.go

```diff
@@ -6,12 +6,12 @@ import (
 	"net"
 	"os"
 	"path/filepath"
-	"time"
+	"sync"
 
 	"github.com/NebulousLabs/Sia/build"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/persist"
-	"github.com/NebulousLabs/Sia/sync"
+	siasync "github.com/NebulousLabs/Sia/sync"
```

Comment:
We probably don't need safelocks in the gateway anymore, you can replace them with standard mutexes.